packages/forma-36-react-components/package.json

```diff
@@ -3,35 +3,68 @@
   "description": "Contentful UI Component Library",
   "version": "3.67.4",
   "main": "dist/index.js",
-  "module": "dist/esm/index.js",
+  "module": "dist/forma-36-react-components.esm.js",
   "style": "dist/styles.css",
   "license": "MIT",
   "types": "dist/index.d.ts",
   "files": [
-    "package.json",
-    "dist"
+    "dist",
+    "src"
```

Comment:
Nice 👍

Reply:
Do we want to have `src` files in the npm package?

Reply:
Tbh I had the same thought. It's the tsdx default, but I'm not 100% sure why they add it.

Reply:
None of our experience components have it: https://github.com/contentful/experience-packages/blob/master/packages/experience-components/package.json, so I'd not include it here too.